### PR TITLE
Adjust priorities of Minion tasks and enable use of "fast lane"

### PR DESCRIPTION
**Pull request by Martchus**
* See particular commit message for details (it is maybe also easier to review the commits individually)
* Related ticket: https://progress.opensuse.org/issues/89224#note-7

### CODE REVIEW

**Comment by kraih** [comment]
The feature is on the way to factory now. https://build.opensuse.org/request/show/878032

**Review by kraih — APPROVED** [review]
The proposed changes make sense to me.

File: lib/OpenQA/WebAPI/Command/gru/run.pm [hunk 1]
```diff
@@ -24,7 +24,7 @@ has usage       => sub { shift->extract_usage };
 sub run {
     my ($self, @args) = @_;
 
-    getopt \@args, 'o|oneshot' => \my $oneshot, 'reset-locks' => \my $reset_locks;
+    getopt \@args, ['pass_through'], 'o|oneshot' => \my $oneshot, 'reset-locks' => \my $reset_locks;
```
Comment (kalikiana) [inline lib/OpenQA/WebAPI/Command/gru/run.pm:27]: I don't understand what this does. The commit says "Allow passing CLI options to minion" but nothing else is being added there. That's the only change in the commit besides the copyright 🤔

Comment (kraih) [reply lib/OpenQA/WebAPI/Command/gru/run.pm:27]: Looks a bit weird, but is very sensible here. It's an option of the underlying param parsing module that leaves unknown flags in `@args` to be processed somewhere else. https://docs.mojolicious.org/Mojo/Util#getopt

Comment (kraih) [reply lib/OpenQA/WebAPI/Command/gru/run.pm:27]: I'm actually not sure why it's missing, should have been there already.

Comment (Martchus) [reply lib/OpenQA/WebAPI/Command/gru/run.pm:27]: The commit message means that the change allows to *pass* options through "this layer of abstraction". So additional `@args` which go in are passed to the run function of the base class which then handles them (whatever they are). So you can e.g. specify `--jobs 10` and these args are passed to the Minion worker with no further handling from our side.

Comment (kalikiana) [reply lib/OpenQA/WebAPI/Command/gru/run.pm:27]: Thank you for clarifying! Confusion resolved. 👍🏾

**Review by kalikiana — APPROVED** [review]

**Pull request merged**